docs/manufacturer/manufacturer.md

```diff
@@ -1,8 +1,14 @@
 ---
-sidebar_position: 2
+sidebar_position: 1
 sidebar_label: Manufacturer
 ---
 
-# Manufacturer
+# Welcome to the Manufacturer section
```

Comment:
Please use  Title Case. As this project is still very young we have to update ... our own documentation 😛 

```suggestion
# Welcome to the Manufacturer Section
```